packages/capacitor/src/hooks/useToast.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react'
+import { toast } from 'react-hot-toast'
+
+type ShowToastProps = {
+  message: string
+  color: string
+  bgColor: string
+  duration?: number
+}
+
+const useToast = () => {
+  const showSuccessToast = useCallback(
+    ({ message, color, bgColor, duration = 4000 }: ShowToastProps) => {
+      return toast.success(message, {
+        position: 'bottom-center',
+        style: {
+          backgroundColor: bgColor,
+          color,
+        },
+        duration,
+      })
+    },
+    []
+  )
+
+  const showErrorToast = useCallback(
+    ({ message, color, bgColor, duration = 4000 }: ShowToastProps) => {
+      return toast.error(message, {
+        position: 'bottom-center',
+        style: {
+          backgroundColor: bgColor,
+          color,
+        },
+        duration,
+      })
+    },
+    []
+  )
+
+  return { showSuccessToast, showErrorToast }
+}
```

Comment:
REQ: 99% of this code is the same. I would also expose the position prop, that's what makes it reusable